### PR TITLE
Adjust publish NavLink inubekit repo

### PR DESCRIPTION
**Pull request by Andresbl123**
ℹ  info      New Release Notes
 #### ⚠️ Pushed to `main`

- refactor: refactor NavLink component to export it as a named export (carlos3k11@gmail.com)
- refactor: remove React and styled-component from the distributable (carlos3k11@gmail.com)
- refactor(NavLink): adjust imports (carlos3k11@gmail.com)
- feat(NavLink): implement stories codumentation (carlos3k11@gmail.com)
- feat: implement the NavLink component (carlos3k11@gmail.com)
- feat: adjust work with js files (carlos3k11@gmail.com)
- feat: implement necessary dependencies (carlos3k11@gmail.com)
- refactor: remove unused dependencies (carlos3k11@gmail.com)
- feat(Storybook): disable telemetry and update documentation settings (carlos3k11@gmail.com)
- feat: update component name in package.json (carlos3k11@gmail.com)
- Initial commit ([@Andresbl123](https://github.com/Andresbl123))

#### Authors: 2

- Andres ([@Andresbl123](https://github.com/Andresbl123))
- Andres Babativa (carlos3k11@gmail.com)

### CODE REVIEW

File: src/NavLink/styles.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+`;
+
+const StyledLink = styled(Link)`
+  box-sizing: border-box;
+  text-decoration: none;
+  width: 100%;
+  cursor: ${({ disabled }) => disabled && "not-allowed"};
+`;
+
+export { StyledLink, StyledNavList };
```
Comment (JanuarAsprilla) [inline src/NavLink/styles.js:60]: Por favor `export` usar la palabra clave en cada const Styled

File: src/NavLink/stories/NavLink.Controller.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  };
+
+  return (
+    <ul>
+      <NavLink {...props} selected={select} onClick={onClick} />
+    </ul>
+  );
+};
+
+export { NavLinkController };
```
Comment (JanuarAsprilla) [inline src/NavLink/stories/NavLink.Controller.tsx:18]: exportar en su funcion directamente 
ejemplo `export const NavLinkController`

Comment (Andresbl123) [reply src/NavLink/stories/NavLink.Controller.tsx:18]: Resuelto

File: src/NavLink/styles.js (hunk 1 above)
Comment (Andresbl123) [reply src/NavLink/styles.js:60]: resuelto

**Review by JanuarAsprilla — APPROVED** [review]

**Pull request merged**